Dockerfile

```diff
@@ -33,7 +33,18 @@ RUN cd /monitors/ \
  && terraform init -input=false || terraform init -input=false -upgrade \
  && rm -rf terraform-sumologic-sumo-logic-monitor
 
-ARG BUILD_TAG=latest
-ENV TAG=$BUILD_TAG
+ ARG BUILD_TAG=latest
+ ENV TAG=$BUILD_TAG
+ 
+ LABEL name="Sumo Logic Kubernetes Setup" \
+     vendor="Sumo Logic" \
+     version="${BUILD_TAG}" \
+     release="1" \
+     summary="$SUMMARY" \
+     description="$DESCRIPTION" \
+     maintainer="opensource-collection-team@sumologic.com"
```

Comment:
I was under the impression these were openshift-specific labels. Why are we adding them to the non-ubi Dockerfile?

Reply:
I can remove them. I think most of them are informative

Reply:
removed